src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1055,30 +1070,57 @@ public override Task<object> ExecuteScalarAsync(CancellationToken cancellationTo
         /// Unlike the ADO.NET method which it replaces, this method returns a Npgsql-specific
         /// DataReader.
         /// </remarks>
-        /// <returns>A DbDataReader object.</returns>
-        public new NpgsqlDataReader ExecuteReader(CommandBehavior behavior) => (NpgsqlDataReader) base.ExecuteReader(behavior);
+        /// <returns>A NpgsqlDataReader object.</returns>
+        public new NpgsqlDataReader ExecuteReader(CommandBehavior behavior)
+            => (NpgsqlDataReader) base.ExecuteReader(behavior);
```

Comment:
Nit: remove space

Reply:
Also, let's take this change to call our private ExecuteDbDataReader directly, instead of passing through base.ExecuteReader. This will remove a bit of useless virtual indirection - we're already doing this for Execute{Scalar,NonQuery}. Let's do it for all ExecuteReader overloads.